packages/@aws-cdk/aws-glue/README.md

```diff
@@ -69,6 +70,22 @@ new glue.Job(this, 'PythonSparkStreamingJob', {
 });
 ```
 
+Note that Streaming Jobs are also available in Scala:
+
+```ts
+new glue.Job(this, 'ScalaSparkStreamingJob', {
+  executable: glue.JobExecutable.scalaStreaming({
+    glueVersion: glue.GlueVersion.V2_0,
+    script: glue.Code.fromBucket(bucket, 'src/com/example/HelloWorld.scala'),
+    className: 'com.example.HelloWorld',
+    extraJars: [glue.Code.fromBucket(bucket, 'jars/HelloWorld.jar')],
+  }),
+  description: 'an example Scala Streaming job',
+  defaultArguments: {"VAL1":"value1"},
```

Comment:
The two examples have different fields: is there a particular reason why one should have `defaultArguments` and the other not?